src/hooks/useDragAndDrop/types.ts

```diff
@@ -0,0 +1,15 @@
+type DragAndDropParams = {
+    dropZone: React.MutableRefObject<HTMLDivElement | null>;
+    onDrop?: (event: DragEvent) => void;
+    shouldAllowDrop?: boolean;
+    isDisabled?: boolean;
+    shouldAcceptDrop?: (event: DragEvent) => boolean;
+};
+
+type DragAndDropOptions = {
```

Comment:
```suggestion
type DragAndDropResult = {
```

`DragAndDropOptions` gives the impression that is a type for supplying options to `useDragAndDrop`, which is not the case here.

Reply:
Yeah, make sense. I just moved that code but it's good idea to rename it